recipes/foxglove-websocket/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class FoxgloveWebSocketConan(ConanFile):
+    name = "foxglove-websocket"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/foxglove/ws-protocol"
+    description = "A C++ server implementation of the Foxglove WebSocket Protocol"
+    license = "MIT"
+    topics = ("foxglove", "websocket")
+
+    settings = ("os", "compiler", "build_type", "arch")
+    requires = ("nlohmann_json/[^3.10.4]", "websocketpp/[^0.8.2]")
```

Comment:
We avoid version ranges in CCI... it was not documented so I tried to note done the motivation for this in a new pr https://github.com/conan-io/conan-center-index/pull/8831

 ```suggestion
    requires = ("nlohmann_json/3.10.4", "websocketpp/0.8.2")
```